test/fuzz/luaL_loadbuffer/serializer.h

```diff
@@ -19,6 +19,7 @@ using namespace lua_grammar;
 
 /**
  * Fuzzing parameters:
+ * kMaxCounterValue - value in `if counter > kMaxCounterValue then break end`
```

Comment:
Nit: Missed dot at the end of the sentence.

Reply:
Fixed